lk2nd/device/dts/msm8909/msm8909-qrd-skub.dts

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8909 0>;
+	qcom,board-id = <QCOM_BOARD_ID_QRD 0x09>,
+			<QCOM_BOARD_ID(QRD, 1, 0) 0x09>;
+};
+
+&lk2nd {
+	lxf_p5001_b01 {
+		model = "Lenovo Yoga Tab 3 10 LTE";
+		compatible = "lenovo,lxf-p5100-b01";
+		lk2nd,match-panel;
+
+		panel {
+			compatible = "lenovo,lxf-p5100-b01-panel";
```

Comment:
```suggestion
			compatible = "lenovo,lxf-p5100-b01-panel", "lk2nd,panel";
```

Reply:
alright  I have thought in to it a bit I will leave a cmment about in the code